hack/build-rpm-release.sh

```diff
@@ -57,7 +57,19 @@ make clean
 
 # migrate the tito artifacts to the Origin directory
 mkdir -p "${OS_OUTPUT}"
-mv "${tito_output_directory}"/* "${OS_OUTPUT}"
+# mv exits prematurely with status 1 in the following scenario: running as root,
```

Comment:
TBH this is a bunch of really complicated work around for a scenario that, while annoying, isn't *that* common.  Going to merge, but it increases maintenance overhead for everyone else in the future.